src/main/java/org/spongepowered/api/block/BlockType.java

```diff
@@ -50,7 +51,7 @@
      *
      * @return The id
      */
-    String getId();
+    String getName();
```

Comment:
CatalogType has the getId method, this is simple overriding the getName method to replace the javadoc.